tests/utils/index.js

```diff
@@ -1,239 +1,86 @@
 'use strict';
 
-const fs = require('fs');
-const os = require('os');
-const path = require('path');
-const crypto = require('crypto');
-const BbPromise = require('bluebird');
-const fse = require('fs-extra');
-const execSync = require('child_process').execSync;
-const AWS = require('aws-sdk');
-
-// mock to test functionality bound to a serverless plugin
-class ServerlessPlugin {
-  constructor(serverless, options, testSubject) {
-    this.options = options;
-    this.serverless = serverless;
-
-    Object.assign(
-      this,
-      testSubject
-    );
-  }
-}
-
-const serverlessExec = path.join(__dirname, '..', '..', 'bin', 'serverless');
-
-const getTmpDirPath = () => path.join(os.tmpdir(),
-  'tmpdirs-serverless', 'serverless', crypto.randomBytes(8).toString('hex'));
-
-const getTmpFilePath = (fileName) => path.join(getTmpDirPath(), fileName);
-
-const replaceTextInFile = (filePath, subString, newSubString) => {
-  const fileContent = fs.readFileSync(filePath).toString();
-  fs.writeFileSync(filePath, fileContent.replace(subString, newSubString));
-};
+const awsCleanup = require('./aws-cleanup');
```

Comment:
I wonder do we really need such index file (?) It always imposes some extra maintance cost.

Does it make a real difference over requiring specific utils in target tests directly?

Reply:
I like that idea. another benefit is that it's way easier to reason about the imports when not importing via `index.js`.